packages/functional-tests/pages/login.ts

```diff
@@ -628,12 +634,17 @@ export class LoginPage extends BaseLayout {
   async getConfig() {
     // Content server config is stored in the `meta` tag of the html page.
     // We can check this tag to see if the content server has enabled this React feature flag.
-    await this.goto();
-    const metaConfig = this.page.locator(
+    // fxa-content-server config
```

Comment:
Stray comment?

Reply:
The first check comes from content-server (when root div is `#stage`). I'm not sure why, but we're naming this differently in React (`#root`). This comment is to differentiate content-server config vs fxa-settings config.